File: Android/02.After/usecase/src/main/java/jp/nuits/hatpepper/usecase/Shop.kt
package jp.nuits.hatpepper.usecase

/**
 * 店舗
 * @property id ID
 * @property name 名称
 */
class Shop(
    val id : String,
    val name : String,
    val genre : String,
    val genreCatch : String,
    val photoUrl : String,
    val budget : String,
    val access : String) {
}

File: HatPepper/03.DDD/HatPepper.Search.Gateway.HotPepper/Shop.cs

using System.Text.Json.Serialization;

namespace HatPepper.Search.Gateway.HotPepper;

/// <summary>
/// 店舗
/// </summary>
/// <param name="Name">名称</param>
/// <param name="Genre">ジャンル</param>
public record Shop(
    [property: JsonPropertyName("id")] string Id,
    [property: JsonPropertyName("name")] string Name,
    [property: JsonPropertyName("genre")] Genre Genre);